the-scheduled-lambda/python/the_scheduled_lambda/the_scheduled_lambda_stack.py

```diff
@@ -13,16 +13,16 @@ def __init__(self, scope: core.Construct, construct_id: str, **kwargs) -> None:
         super().__init__(scope, construct_id, **kwargs)
 
         # DynamoDB Table
-        table = dynamo_db.Table(self, "Table",
+        table = dynamo_db.Table(self, "RequestTable",
                                 partition_key=dynamo_db.Attribute(name="requestid", type=dynamo_db.AttributeType.STRING),
                                 removal_policy=core.RemovalPolicy.DESTROY
                                 )
 
         # Create the Lambda function we want to run on a schedule
         scheduled_lambda = _lambda.Function(self, 'ScheduledLambda',
-                                            runtime=_lambda.Runtime.NODEJS_12_X,
-                                            code=_lambda.Code.from_asset('lambda-fns/scheduled-lambda'),
```

Comment:
I moved the function to lambda_fns (python standard) instead lambda-fns/scheduled-lambda. There is only one function, so there is no need to use many folders.